src/initial_corpus/dependency_graph/normalize.rs

```diff
@@ -246,6 +254,34 @@ mod tests {
             },
             ParameterNormalization::new("widget_id", Some("countries"))
         );
+        assert_eq!(
+            ParameterNormalization {
+                name: "pet-id",
+                normalized: "pet|id".into(),
+            },
+            ParameterNormalization::new("pet-id", Some("pets"))
+        );
+        assert_eq!(
+            ParameterNormalization {
+                name: "petId",
+                normalized: "pet|id".into(),
+            },
+            ParameterNormalization::new("petId", Some("pets"))
+        );
+        assert_eq!(
+            ParameterNormalization {
+                name: "petid",
+                normalized: "pet|id".into(),
+            },
+            ParameterNormalization::new("petid", Some("pet"))
+        );
+        assert_eq!(
+            ParameterNormalization {
+                name: "PetID",
+                normalized: "pet|id".into(),
+            },
+            ParameterNormalization::new("PetID", Some("pet"))
+        );
```

Comment:
I am trying to understand why some tests use plural and some use singular pet(s)?